Feature/#507 separate package long desc with readme (#1284)
Avaiga/taipy: package_desc.md

Unless required by applicable law or agreed to in writing, software distributed under the
License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
either express or implied. See the License for the specific language governing permissions
and limitations under the License.

## ⭐️ What is Taipy

Taipy is an open-source Python library for easy, end-to-end application development for data
scientists and machine learning engineers, featuring what-if analyses, smart pipeline execution,
built-in scheduling, and deployment tools.

- Enables building production-ready web applications.
- No need to learn new languages or full-stack frameworks.
- Concentrate on Data and AI algorithms without development and deployment complexities.

- Website: https://www.taipy.io
- Documentation: https://docs.taipy.io/en/latest/
- Demos & Examples: https://docs.taipy.io/en/latest/gallery/
- Source code: https://github.com/Avaiga/taipy/
- Contributing: https://docs.taipy.io/en/latest/contributing/contributing/
- Bug reports: https://github.com/Avaiga/taipy/issues
- Discord support: https://discord.com/invite/SJyz2VJGxV

## ✨ Features

> - **Python-Based UI Framework:** Taipy is designed for Python users, particularly those working in AI and data science. It allows them to create full stack applications without needing to learn additional skills like HTML, CSS, or JavaScript.

Reviewer: ```suggestion
- **Python-Based UI Framework:** Taipy is designed for Python users, particularly those
  in AI and data science. It allows them to create full-stack applications without
  learning additional skills like HTML, CSS, or JavaScript.
```

> - **Pre-Built Components for Data Pipelines:** Taipy includes pre-built components that allow users to interact with data pipelines, including visualization and management tools.

Reviewer: ```suggestion
- **Pre-Built Components for Data Pipelines:** Taipy includes pre-built components that
  allow users to interact with data pipelines, including visualization and management
  tools.
```

> - **Scenario and Data Management Features:** Taipy offers features for managing different business scenarios and data, which can be useful for applications like demand forecasting or production planning.

Reviewer: ```suggestion
- **Scenario and Data Management Features:** Taipy offers features for managing different
  business scenarios and data, which can be useful for applications like demand forecasting
  or production planning.
```

> - **Version Management and Pipeline Orchestration:** It includes tools for managing application versions, pipeline versions, and data versions, which are beneficial for multi-user environments.

Reviewer: ```suggestion
- **Version Management and Pipeline Orchestration:** It includes tools for managing
  application versions, pipeline versions, and data versions, which are beneficial 
  for multi-user environments.
```
```suggestion
- **Version Management and Pipeline Orchestration:** It includes tools for managing application versions, pipeline versions, and data versions, which are beneficial for multi-user environments.
```

## Installation

To install Taipy stable release run:
```bash
pip install taipy
```

> To install Taipy on a Conda Environment or from source, please refer to the [Installation Guide](https://docs.taipy.io/en/latest/installation/).<br />

Reviewer: ```suggestion
To install Taipy on a Conda Environment or from source code, please refer to the
[Installation Guide](https://docs.taipy.io/en/latest/installation/).<br />
```

To get started with Taipy, please refer to the [Getting Started Guide](https://docs.taipy.io/en/latest/getting_started/).

## Contributing

Writing code isn’t the only way to contribute to Taipy. You can also:
- help us stay on top of new and old issues
- develop tutorials, presentations, and other educational materials
- maintain and improve [our website](https://www.taipy.io) and [documentation](https://docs.taipy.io/en/latest/)
- review pull requests
- help with outreach and onboard new contributors

For more information about the ways you can contribute to Taipy, visit our
[Contributing Guide](https://docs.taipy.io/en/latest/contributing/contributing/).
If you’re unsure where to start or how your skills fit in, reach out on Discord. You can also ask on
GitHub, by opening a new issue or leaving a comment on a relevant issue that is already open.

If you are new to contributing to open source projects, [this guide](https://opensource.guide/how-to-contribute/)

> helps explain why, what, and how to successfully get involved.

Reviewer: Is this necessary on Pypi?

Reviewer 2: I agree; I think Pypi users want a description, information on license, security and maintenance, and a link to the documentation. Since you can't contribute on Taipy, a contribution section doesn't make much sense.

Author: I think we can keep the link to the contribution page, so that we encourage people to contribute to Taipy, including reporting any bug they found when installing and using Taipy

## Code of conduct

Taipy is an open source project developed by the Taipy development team and a community of
[contributors](https://docs.taipy.io/en/latest/contributing/contributors/). Please check out the
[Taipy Code of Conduct](https://docs.taipy.io/en/latest/contributing/code_of_conduct/) for guidance

> on how to interact with others in a way that makes our community thrive.

Reviewer: Is this necessary on Pypi?

Author: A lot of other packages are also showing a link to their code_of_conduct page. I think it's a common practice
